tests/regression_results/PRD.json

```diff
@@ -3,12 +3,12 @@
     "coords": {
         "dim_average_electron_density": {
             "attrs": {},
-            "data": 25.0,
+            "data": 29.363636363636363,
             "dims": []
         },
         "dim_average_electron_temp": {
             "attrs": {},
-            "data": 9.137931034482758,
+            "data": 8.636363636363637,
             "dims": []
         },
```

Comment:
Why does the PRD point move? It should be in H-mode, and therefore shouldn't be affected by this change